verification/src/http_server/routers/app.rs

```diff
@@ -21,10 +25,28 @@ impl AppRouter {
     }
 }
 
+#[derive(OpenApi)]
+#[openapi(
+    handlers(
+        status::status,
+        solidity::multi_part::verify,
+    ),
+    components(
+        VerificationResponse,
+        MultiPartFiles,
+        VerificationResult,
+        VerificationStatus,
+    ),
```

Comment:
Another inconvenience here is that we have to break abstractions and collect all endpoints here.
I don't see any possibility to handle it through configs